ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -703,6 +705,7 @@ private void createBuilderTask(List<Task<?>> rootTasks) {
   private int executeIncrementalLoad(long loadStartTime) throws Exception {
     // If replication policy is changed between previous and current repl load, then drop the tables
     // that are excluded in the new replication policy.
+    this.childTasks = (this.childTasks == null) ? new ArrayList<>() : this.childTasks;
```

Comment:
How much of this refactor needed for the specific change?
We might want to do it in a separate PR so we can backport the fix without the conflicts generated by the refactor when needed